rfcs/0125-bootspec.md

```diff
@@ -135,26 +137,39 @@ Using the following JSON:
       "loglevel=4"
     ],
 
-    // The label of the system. It should contain the operating system, kernel version,
+    // (Required) The label of the system. It should contain the operating system, kernel version,
     // and other user-relevant information to identify the system. This corresponds
     // loosely to `config.system.nixos.label`.
     "label": "NixOS 21.11.20210810.dirty (Linux 5.15.30)",
 
-    // Top level path of the closure, in case some spelunking is required
+    // (Required) Top level path of the closure, in case some spelunking is required
     "toplevel": "/nix/store/xxx-nixos-system-xxx",
 
+    // (Optional) Mapping of specialisation names to their bootspec document
     "specialisation": {
       // <name> corresponds to <name> in specialisation.<name>.configuration.
       // Note that it is not valid for a specialisation to be a different version than the main document.
       "<name>": {
         // A full Bootspec v1 document, _without_ the `"v1"` key.
         // Note that it is not valid for a specialisation to have further specialisations.
       }
+    },
+
+    // (Optional) User-specified metadata
+    "extensions": {
+      // <namespace> corresponds to a user-provided namespace
+      // to reduce the chances of collision when using
+      // multiple extensions at once
```

Comment:
Nit: The reflowing looks a little weird here, when compared to the line lengths of the other comments (e.g. compared to https://github.com/grahamc/rfcs/pull/7/files#diff-709266622ec29f9cabb5a5b4e6f93792851db9cfebd72bdf78f3032b437d8764L148).